velox/exec/tests/utils/PlanBuilder.cpp

```diff
@@ -1478,6 +1478,94 @@ PlanBuilder& PlanBuilder::window(
       sortingOrders,
       windowNames,
       windowNodeFunctions,
+      false,
+      planNode_);
+  return *this;
+}
+
+PlanBuilder& PlanBuilder::streamingWindow(
```

Comment:
This logic looks the same as that in window() PlanNode except for inputsSorted boolean. Please refactor to use a common function with a boolean for inputsSorted as a parameter.